content/rs/installing-upgrading/configuring/cluster-lba-setup.md

```diff
@@ -86,8 +86,7 @@ An additional setting can be done to allow (on average) closer termination of cl
 
 ### RS database configuration
 
-After the cluster settings are updated and the LBs are configured,
-you can go to the RS admin console at <https://load-balancer-virtual-ip:8443/> and [create a new database]({{< relref "/rs/administering/creating-databases/_index.md" >}}).
+After the cluster settings are updated and the LBs are configured you can create a new Active-Active database with the `crdb-cli` utility. See the ['crdb-cli' reference]({{<relref "/rs/references/crdb-cli-reference.md">}}) for more information about creating Active-Active databases from the command line.
```

Comment:
```suggestion
After the cluster settings are updated and the LBs are configured, you can create a new Active-Active database using the `crdb-cli` utility. See the ['crdb-cli' reference]({{<relref "/rs/references/crdb-cli-reference.md">}}) for more information about creating Active-Active databases from the command line.

Reply:
Kaitlyn, note also, that the sentence at line 19 is grammatically incorrect: 

When DNS cannot be used, clients can still connect to the endpoints with the IP addresses, but the benefits of load balancing and **automatically updates IP address** are missing.